commit 5e0962474cbed1dbfea34afd8eeb6cd71dc518d3
Author: Carl George <carl@george.computer>
Date:   Tue Apr 5 18:49:13 2022 -0500

    BUILD(cmake): Find and link Poco::XML
    
    Without this, building against the latest poco snapshot results in
    errors like:
    
    /usr/bin/ld: /builddir/build/BUILD/mumble-1.4.230.src/src/mumble/PluginManifest.cpp:72: undefined reference to `typeinfo for Poco::XML::Element'
    
    (cherry picked from commit fefdd79ebcd53a3035967789d004938ee39e2030)
    
    # Conflicts:
    #       src/mumble/CMakeLists.txt

diff --git a/src/mumble/CMakeLists.txt b/src/mumble/CMakeLists.txt
index 4530dc753..be37fd674 100644
--- a/src/mumble/CMakeLists.txt
+++ b/src/mumble/CMakeLists.txt
@@ -384,11 +384,16 @@ target_include_directories(mumble
 		"${PLUGINS_DIR}"
 )
 
-find_pkg(Poco COMPONENTS Zip)
+find_pkg(Poco
+	COMPONENTS
+		XML
+		Zip
+)
 
 if(TARGET Poco::Zip)
 	target_link_libraries(mumble
 		PRIVATE
+			Poco::XML
 			Poco::Zip
 	)
 else()
